app/assets/stylesheets/base/_typography.scss

```diff
@@ -0,0 +1,50 @@
+// Body Typography
+//
+// Mixin to adjust the font size and line height only. Typically used for one off elements that do not require any top
+// or bottom margins.
+
+@mixin body($font-size, $line-height) {
+  @if $responsive == true {
+    font-size: ($font-size / 16) + rem;
+    line-height: ($line-height / 16) + rem;
+  } @else {
+    font-size: $font-size + px;
+    line-height: $line-height + px;
+  }
+}
+
+%type {
+  @include body(16, 24);
+}
+
+h1,
+h2,
+h3 {
+  font-weight: 500;
+}
+
+a {
+  color: $primary-blue;
+  text-decoration: none;
+
+  &:hover {
+    text-decoration: underline;
+  }
+
+  &:focus {
+    text-decoration: underline;
+    outline: 2px solid $primary-blue-dark;
+  }
+
+  &:visited {
+    color: $primary-orange;
+  }
+
+  &:not([href*="fincap.org.uk"]):after {
```

Comment:
@johnplayer1982 what's going to happen to the styling when the domain changes from "finch.org.uk"?

Reply:
I assumed the production domain will always be fincap.org.uk? Is that not the case?

Reply:
Even if it was we will have to remember to change this part of the CSS when the domain name is changed in the future.

Reply:
Yep, same as the MAS site, we have the same situation there

Reply:
Is it possible to use a variable for the href attribute?